src/Model/Response/ItemPropertiesListResponse.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
```

Comment:
Oh, i just noted we are missing UT for these responses... Could you please add them?